### ISSUE TITLE
Postpone 2.0

### ISSUE THREAD

**Issue by johnsoncodehk** [vuejs/language-tools#3294]
We originally planned to use 1.6 as the last official version of v1, and transition to 2.0 through 1.7 pre-release, but before we started to make more major breaking changes, we accumulated some relatively important updates in 1.7, such as the Volar.js 1.7 upgrade, Vue 3.3 / TS 5.1.3 compatibility, important performance regression fixes, Scripts API, new directive comments syntax support, and more.

Since these updates stuck in pre-release were blocking some downstream projects, I decided to release these updates to 1.8 first and make the remaining 2.0 changes in the 1.9 pre-release. It's also good for me because I no longer need to rush through 2.0 tasks.

But there is two breaking change that has already occurred and it cannot be undone:
- If you use volar.config.js, please note that the `plugins` option has now been renamed to `services`
- #3134
